misc/bazel/internal/git_lfs_probe.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+"""
+Probe lfs files.
+For each source file provided as output, this will print:
+* "local", if the source file is not an LFS pointer
+* the sha256 hash, a space character and a transient download link obtained via the LFS protocol otherwise
```

Comment:
How long is this transient link valid?
We're batch-obtaining all these links, and then downloading them after each other, is there any chance of them expiring while we're still downloading?

Reply:
This is probably configured at the server level, but what we currently have is 1 hour validity, so plenty of time